From 5ceb021eacb06b05620c32d064c966f4928904eb Mon Sep 17 00:00:00 2001
From: Xiaoyu Yao <xyao@apache.org>
Date: Fri, 14 Aug 2015 12:33:43 -0700
Subject: [PATCH 0801/2748] HDFS-8565. Typo in dfshealth.html -
 Decomissioning. (nijel via xyao)

(cherry picked from commit 1569228ec9090823186f062257fdf1beb5ee1781)
(cherry picked from commit 17256e2ae81d0b40493901fc89ca0d987788f69b)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt

Change-Id: I6feacb337ed01bd4420f6f2b1f341d6605dce51c
---
 .../src/main/webapps/hdfs/dfshealth.html           |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
index 007d6e8..79df028 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
@@ -305,7 +305,7 @@
 </table>
 </small>
 
-<div class="page-header"><h1><small>Decomissioning</small></h1></div>
+<div class="page-header"><h1><small>Decommissioning</small></h1></div>
 <small>
 <table class="table">
   <thead>
-- 
1.7.9.5

